src/meson.build

```diff
@@ -0,0 +1,256 @@
+# -------------------------- Configuration
+commit_hash_cmd = run_command('git', 'rev-parse',
+                              '--short', 'HEAD',
+                             check: true)
+commit_hash = commit_hash_cmd.stdout().strip()
+python_getdate = '''
+import datetime
+
+today = datetime.date.today()
+today_str = today.strftime('%Y-%m-%d')
+print(today_str)
+'''
+today_cmd = run_command('python3', '-c',
+                        python_getdate,
+                       check: true)
+today_str = today_cmd.stdout().strip()
```

Comment:
It probably makes sense to check https://reproducible-builds.org/docs/source-date-epoch/ here.

Reply:
Neat, modifying to check the variable